src/main/kotlin/no/nav/helsearbeidsgiver/pdl/domene/DomeneIntern.kt

```diff
@@ -12,7 +12,17 @@ data class FullPerson(
     val navn: PersonNavn,
     val foedselsdato: LocalDate,
     val ident: String? = null,
-)
+    val gradering: String? = null,
+    val geografiskTilknytning: String? = null,
+) {
+    fun getKodeverkDiskresjonskode(): String? {
+        return when (gradering) {
+            "STRENGT_FORTROLIG" -> "SPSF"
+            "FORTROLIG" -> "SPFO"
+            else -> null
+        }
+    }
+}
```

Comment:
Bruker vi gradering til noe annet enn dette? Kan vi heller ha `kodeverkDiskresjonskode` som felt?
Sidenote: Hvis vi vet alle strengene vi bryr oss om, kan vi lage enum 🙏

Reply:
enum blir i neste versjon - beholder koden likt for nå, så blir fritakagp-koden lik :)